modules/040-node-manager/docs/FAQ.md

```diff
@@ -244,17 +244,116 @@ Create a `NodeGroup` with the following parameters:
 ```yaml
 cri:
   type: NotManaged
-operatingSystem:
-  manageKernel: false
 ```
 
 Then put the node under the control of `node-manager`.
 
+## How do I update kernel on nodes?
+
+### Debian-based distros
+
+Deploy `NodeGroupConfiguration` scripts, fill `desired_version` with desired kernel version:
```

Comment:
```suggestion
Deploy `NodeGroupConfiguration` scripts, fill `desired_version` with desired kernel version:
Create a `Node Group Configuration` resource by specifying the desired kernel version in the `desired_version' variable of the shell script (the resource's content parameter):
```

Reply:
the resource's content parameter -> spec.content